builtin/logical/pki/backend.go

```diff
@@ -563,19 +570,23 @@ func (b *backend) periodicFunc(ctx context.Context, request *logical.Request) er
 		return nil
 	}
 
+	backgroundSc := b.makeStorageContext(context.Background(), b.storage)
+	go b.runUnifiedTransfer(backgroundSc)
```

Comment:
Did we want this to run every periodic func? Or was this just for the initial startup, +/- the setting being toggled? We could set a boolean to allow it to run once-ish, invalidating the boolean if the config changes, potentially rerunning it? 

(The config change reload should also probably apply to the revocation queue as well).

Reply:
On net-new revocations, we'll write the revocation directly... So I think this could be fine with less frequency FWIW.

Reply:
The idea was we were gonna run this continuously, but yes a delay between runs would be better. I forgot to port that to this implementation 😨

Reply:
Should be addressed in PR